vllm/config.py

```diff
@@ -242,10 +242,12 @@ def __init__(
         pipeline_parallel_size: int,
         tensor_parallel_size: int,
         worker_use_ray: bool,
+        model_load_batch_size: Optional[int] = None,
```

Comment:
Let's rename this variable to `max_parallel_loading_workers` to make the name more straightforward?